File: javascript/2579-count-total-number-of-colored-cells.js
/**
 * 2579. Count Total Number of Colored Cells
 * https://leetcode.com/problems/count-total-number-of-colored-cells/
 */

/**
 * @param {number} n
 * @return {number}
 */
var coloredCells = function (n) {
  return n === 1 ? 1 : 4 * (n - 1) + coloredCells(n - 1);
};

File: swift/2579-count-total-number-of-colored-cells.swift
/**
 * 2579. Count Total Number of Colored Cells
 * https://leetcode.com/problems/count-total-number-of-colored-cells/
 **/

class Solution {
    func coloredCells(_ n: Int) -> Int {
        return n * n + (n-1) * (n-1)
    }
}